src/database/dbConnection.js

```diff
@@ -0,0 +1,27 @@
+const { Pool } = require('pg');
+const url = require('url');
+
+let dbUrl = process.env.DB_LOCAL_URL;
+if (process.env.NODE_ENV === 'pro') {
+  dbUrl = process.env.DB_HEROKU_URL;
+}
+else if (process.env.NODE_ENV === 'test') {
+  dbUrl = process.env.DB_TESTING_URL;
+}
+else {
+  dbUrl = process.env.DB_LOCAL_URL;
```

Comment:
```suggestion
  dbUrl = process.env.DB_LOCAL_URL;
```
What do I need this sentence has given the variable the same value as the previous one